sound/soc/sof/ipc4-topology.c

```diff
@@ -1480,6 +1480,31 @@ static int sof_ipc4_prepare_process_module(struct snd_sof_widget *swidget,
 	return sof_ipc4_widget_assign_instance_id(sdev, swidget);
 }
 
+static int sof_ipc4_large_config_kpb(struct snd_sof_widget *swidget)
+{
+	/* To be implemented later */
```

Comment:
this is hard to review @libinyang . Im not sure about what this is mean for and I cant see why you need a new op or a walk through for the entire pipeline

Reply:
For some modules, for example KPB, it needs send ipcs: 'initinstance', 'bind' and 'largeconfigset'. And 'largeconfigset' should be called after 'bind'. But in our original code, there is no formal point to call larconfigset. To make 'largeconfigset' to have the same formal flow like 'initinstance' and 'bind', I would like to add the ops in the walk.

Reply:
@libinyang why should the large config be setn after binding?

Reply:
The most situation we met is after binding we need set some exta config. For example, copier can connect 2 sink modules. In the initistance, we will set config for the first sink. After the second sink is bound, we need to set the copier output format for the second connection. Another example is cAVS KPB. When KPB is connected to the sinks, we need to collect all the modules in the QID1 connection and set the information to KPB with largeconfig. 
Of course, some modules can set largeconfig without binding. But to give a formal rule, I prefer to set largeconfig after binding. What do you think?

Reply:
But why even mention **large** in the function names? The 'large' message is a term used for cases in IPC4 when the IPC message have additional payload delivered via mailbox.
The operation is just a generic set the config or update the config of a given widget, component, the API does not need to describe (and thus limit) the means a given operation must be done.

Reply:
OK, I will remove the 'large'